php-exercise2/src/controller/controller.php

```diff
@@ -0,0 +1,34 @@
+<?php 
+require_once 'src/model/model.php';
+use model\model\Operations;
+
+$result = "";
```

Comment:
Please create a class for each of your objects.

Reply:
Done